settings/DevHome.Settings/Strings/en-us/Resources.resw

```diff
@@ -571,10 +571,16 @@
   </data>
   <data name="FileExplorerSourceControlIntegration_Name" xml:space="preserve">
     <value>File Explorer Source Control Integration</value>
-    <comment>Name of experimental feature 'FileExplorer SourceControl Integration' on the 'Settings -> Experiments' page where you enable it.</comment>
+    <comment>Name of experimental feature 'FileExplorer SourceControl Integration' on the 'Settings -&gt; Experiments' page where you enable it.</comment>
   </data>
   <data name="FileExplorerSourceControlIntegration_Description" xml:space="preserve">
     <value>View property information (such as commit status, last commit date etc.) from source control technologies inside File Explorer</value>
-    <comment>Inline description of the FileExplorerSourceControlIntegration experimental feature on the 'Settings -> Experiments' page where you enable it.</comment>
+    <comment>Inline description of the FileExplorerSourceControlIntegration experimental feature on the 'Settings -&gt; Experiments' page where you enable it.</comment>
+  </data>
+  <data name="RepositoryManagementExperiment_Description" xml:space="preserve">

```

Comment:
Not very meaningful. Is the description needed?

Reply:
Looks like the only change was `>` to `-&gt;` in the description and it matches other strings that use `-&gt;`  I'm going to keep this change.